### ISSUE TITLE
ExtendedPropery with a dateformat value will fail to load from saved modelfile.

### ISSUE THREAD

**Issue by larsthelord** [otykier/TabularEditor#687]
**Describe the bug**
When creating extendedProperty on Table. If the value of the extendedProperty is a string containing a datetime, Tabular Editor fails to open this file after it has been saved.
It can however open the same model from server, with the datetime string, without any problems

The following error is returned:
> Error: Cannot convert value 'dd.MM.yyyy HH:mm:dd' to required type 'String'. Check path 'model.tables[x].extendedProperties[x].value

This is a picture how the string containing a datetime is stored in the model file:
![image](https://user-images.githubusercontent.com/22727597/99529486-2d830700-29a0-11eb-852a-96e3465a4cd1.png)

**To Reproduce**

1. Create extendedProperty on Table object
2. set name = "test"
3. set value = "2020-11-18T13:23:25:147Z"
4. save model as model.bim
5. open model from model.bim file ctrl+o
6. select modelfile you just created
7. see error

**Expected behavior**
Tabular Explorer should be able to open model.bim file with a extendedProperty containing a formatted datetime as value.

**Observed behavior**

The following error is returned:
> Error: Cannot convert value '<dd.MM.yyyy HH:mm:dd>' to required type 'String'. Check path 'model.tables[x].extendedProperties[x].value

**Application specifics**
Tabular Editor version 2.13.0
Azure Analysis Services
Model Compatibility level 1500

**Additional context**

Tabular Editor seems to parse the value as [System.DateTime] when it should just be seen as a [System.String]

Tabular Editor is able to open the same model-object from the server without encountering this error.

**Comment by otykier** [comment]
I was not able to reproduce this. Can you provide a minimal Model.bim file that produces the error when loaded?

**Comment by larsthelord** [comment]
I have included a model with one table.
on that table it has an extendedProperty with the value 1900-01-01T00:00:00.000Z
When i open this file "Example.bim", I get the following error
![image](https://user-images.githubusercontent.com/22727597/99579642-bfaa0000-29de-11eb-801f-5321179c4b09.png)

I had to zip the file to be allowed to upload it
[Example.zip](https://github.com/otykier/TabularEditor/files/5562355/Example.zip)

**Comment by otykier** [comment]
Thanks for getting back to me.

I wrote the following C# application using only the [AMO library](https://www.nuget.org/packages/Microsoft.AnalysisServices.retail.amd64/) to deserialize the JSON (this is the same technique Tabular Editor uses to load the JSON), which causes the same error:

![image](https://user-images.githubusercontent.com/8976200/99598039-37852400-29f9-11eb-9d22-f6cbdd365f0b.png)

So in other words, this is not a bug in Tabular Editor but in the underlying AMO library provided by Microsoft. I will report the bug to them. Maybe you can use a JsonExtendedProperty as a workaround in the mean while?
